UPGRADE.md

```diff
@@ -0,0 +1,37 @@
+# Upgrade guide to Smooch
+
+SupportKit is now Smooch!
+
+With Smooch, the internals of the SDK have changed. It is now using our [public API](http://docs.supportkit.io/rest/) and as such we've bumped the major version to 2.0.
+
+## CDN
+
+To get the new version, you should now include this script in your page :
+
+```html
+<script src="https://cdn.smooch.io/smooch.min.js"></script>
+```
+
+You can still use the old CDN link, but that package won't get updated. It will stay in v1.0.
+
+## Packages
+
+The package is now named `smooch` on npm and bower.
+
+```
+$ npm install smooch
+
+$ bower install smooch
+```
+
+## API
+
+The most notable change is that you no longer use the `SupportKit` global variable, but the new `Smooch` global.
+
+```javascript
+var Smooch = require('smooch');
+
+Smooch.init({appToken: 'your_app_token'});
+```
+
+The rest is pretty much the same! We didn't change the CSS classes, so your customization should be safe.
```

Comment:
I'm hesitant about not changing those. those will be the only sk remaining in the web sdk